src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -99,6 +99,49 @@ public void multipleFromTest() throws IOException {
         Assert.assertEquals(14, getTotalHits(response));
     }
 
+    @Test
+    public void selectAllAndSpecificFieldTest() throws IOException {
+        String[] arr = new String[] {"account_number", "firstname", "address", "birthdate", "gender", "city", "lastname", "balance", "employer", "state", "age", "email", "male"};
+        Set<String> expectedSource = new HashSet<>(Arrays.asList(arr));
+        JSONObject response = executeQuery(String.format(Locale.ROOT,
+                "SELECT *, age FROM %s/account", TestsConstants.TEST_INDEX_BANK));
```

Comment:
np: maybe add LIMIT with a small number like 5, not to iterate through all the documents for this test.

Reply:
Good idea- will add now